Tests/SkillFunctionalTests/Common/HostBot.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace SkillFunctionalTests.Common
+{
+    public enum HostBot
+    {
+        /// <summary>
+        /// Simple host implemented using dotnet 3.1.
+        /// </summary>
+        SimpleHostBotDotNet,
```

Comment:
Could we rename the bots here to match the names used in the pipelines?
```suggestion
        SimpleHostBotDotnet,
```

Reply:
We are standardizing on DotNet (captial N), closing